package.json

```diff
@@ -28,8 +28,7 @@
     "prepare": "husky install"
   },
   "resolutions": {
-    "@types/react": "^18",
-    "@types/react-dom": "^18"
+    "@types/react": "^17.0.2"
```

Comment:
Why did you downgrade here ?

Reply:
pre-upgrade changes, shouldn't be needed anymore

Reply:
I'll change it back